base/random.jl

```diff
@@ -112,7 +112,34 @@ function srand(r::MersenneTwister, seed::Vector{UInt32})
 end
 
 # MersenneTwister jump
-function randjump(mt::MersenneTwister, jumps::Integer, jumppoly::AbstractString)
+
+"""
+    randjump(r::MersenneTwister, jumps, steps=10^20) -> Vector{MersenneTwister}
+
+Create an array of size `jumps` of initialized `MersenneTwister` RNG
+objects corresponding to different substreams of `r`, using the
+"jump-ahead" method. The first element of this array is the RNG `r`
+given as a parameter; each other RNG element is initialized to a state
+equivalent to what would be that of the previous element if it was
```

Comment:
can you just say "the previous element's state moved forward by `steps` steps" ? what is "equivalent to what would be that of" trying to convey here?